sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/VectorizedRleValuesReader.java

```diff
@@ -340,6 +398,71 @@ public Binary readBinary(int len) {
     throw new UnsupportedOperationException("only readInts is valid.");
   }
 
+  @Override
+  public void skipIntegers(int total) {
+    int left = total;
+    while (left > 0) {
+      if (this.currentCount == 0) this.readNextGroup();
+      int n = Math.min(left, this.currentCount);
+      advance(n);
+      left -= n;
+    }
+  }
+
+  @Override
+  public void skipBooleans(int total) {
+    throw new UnsupportedOperationException("only skipIntegers is supported");
```

Comment:
Do we have a plan to support `skipBooleans`?

Reply:
`only skipIntegers is supported` has a little different meaning from `only skipIntegers is valid`. I'm curious what was the correct intention here.

Reply:
I can change it to `only skipIntegers is valid`. It's because Parquet RLE encoding only support integers which is why all the others are unimplemented.

Reply:
+1 for changing.